site/docs/src/json/applications/search-post-request.json

```diff
@@ -0,0 +1,10 @@
+{
+  "search": {
+    "name": "registration",
+    "active": true,
```

Comment:
I think `state` on the Application is deprecated in favor of the `state` field, should we use that instead? The current Application API response example uses the name `Forum` for reference, so we could use that to keep the theme going.

Reply:
@robotdan are you saying that we should name this request parameter `state`? Or that the query should use `state`?

The existing queries for retrieving all applications use `active` rather than `state`. There are also 3 `ObjectState` values, but applications only use two of them.